core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/drs/DrsHubResolver.scala

```diff
@@ -45,16 +43,17 @@ class DrsHubResolver(drsHubUrl: String)(implicit
     }
   }
 
-  override def drsServiceAccountEmail(drsUrl: String, userInfo: UserInfo): Future[Option[String]] =
+  override def drsSignedUrl(drsUrl: String, userInfo: UserInfo): Future[Option[String]] =
     resolveDrs(drsUrl, userInfo).map { resp =>
       // The email field must remain an `Option` because DRS servers that do not use Bond (HCA, JDR) do not return a service account
       // AEN 2020-09-08 [WA-325]
-      val saEmail: Option[String] = resp.googleServiceAccount.flatMap(_.data.map(_.client_email))
+      // TODO can it be not an Option now?
```

Comment:
does the DRS spec say anything about this? For instance is it guaranteed that if you ask for an accessUrl and the DRS server doesn't have one, it throws an error instead of returning blank/empty? If there's any chance the response will be null/empty, I think it's best to leave it as an `Option`

Reply:
Can't figure out where to find this in the DRS spec, but experimenting I did experience cases both of an error AND off a null accessURL.

Reply:
The relevant part of the DRS specification is [here](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.5.0/docs/#tag/Objects/operation/GetAccessURL).
In short, if this endpoint responds with an HTTP code 200, the signed URL will be present, for another other return code, it will not be.

However, DRSHub is an intermediary between the DRS server and the end client, and the behavior of DRSHub would need to be examined to understand the responses it provides.

In any case, if you don't get a signed URL back, the submission should fail, and the host/provider should be identified in the error message.